$OpenBSD: patch-gegl_meson_build,v 1.4 2021/03/28 08:02:15 ajacoutot Exp $

Fix non deterministic build:
../gegl-0.4.18/gegl/opencl/gegl-cl-color.c:32:10: fatal error: 'opencl/colors.cl.h' file not found
#include "opencl/colors.cl.h"
         ^~~~~~~~~~~~~~~~~~~~
1 error generated.

Index: gegl/meson.build
--- gegl/meson.build.orig
+++ gegl/meson.build
@@ -93,6 +93,7 @@ gegl_ldflags = os_osx ? ['-framework', 'OpenCL'] : []
 
 gegl_lib = library(api_name,
   gegl_sources,
+  opencl_headers,
   include_directories: [rootInclude, geglInclude],
   dependencies: [
     babl,
